README.md

```diff
@@ -51,5 +51,5 @@ NOTE: This blog post isn... </p> <br/> <br/>
 [![Discord](https://img.shields.io/badge/-DISCORD-informational?style=for-the-badge&logo=discord&logoColor=23272a&color=7289da)](https://discord.gg/4EDmwmzXzN)
 [![Dev.to](https://img.shields.io/badge/dev.to-0A0A0A?style=for-the-badge&logo=dev.to&logoColor=white)](https://dev.to/kira272921)
 [![Hashnode](https://img.shields.io/badge/Hashnode-2962FF?style=for-the-badge&logo=hashnode&logoColor=white)](https://kirablog.hashnode.dev/)
-[![Twitter](https://img.shields.io/badge/<handle>-%231DA1F2.svg?style=for-the-badge&logo=Twitter&logoColor=white)](https://twitter.com/Kira_272921)
+[![Twitter](https://img.shields.io/badge/Twitter-%231DA1F2.svg?style=for-the-badge&logo=Twitter&logoColor=white)](https://twitter.com/Kira_272921)
```

Comment:
```suggestion
[![Twitter](https://img.shields.io/badge/Kira_272921-%231DA1F2.svg?style=for-the-badge&logo=Twitter&logoColor=white)](https://twitter.com/Kira_272921)
```
Also this could be a solution for this. But I think due its also Hashnode etc. Twitter is the better way.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
As all the other social media badges have their platform name on them instead of my username, so it would be better to have Twitter badge's name as "Twitter" instead of "Kira_272921"